kolena/_experimental/search/_internal/datatypes.py

```diff
@@ -24,5 +28,21 @@ class LocatorEmbeddingsDataFrameSchema(pa.SchemaModel):
 
     embedding: Series[pa.typing.String] = pa.Field(coerce=True)
     """
-    Embedding vector (`np.ndarray`) corresponding to a searchable representation of the sample.
+    Embedding vector (base64-encoded string of `np.ndarray`) corresponding to a searchable representation of the sample.
+    """
+
+
+class DatasetEmbeddingsDataFrameSchema(pa.SchemaModel):
```

Comment:
Do we apply any validation for this schema?

Reply:
Could you elaborate the question? This schema is [used to validate the df](https://github.com/kolenaIO/kolena/pull/475/files#diff-1dce437eca8dea142eb32b0f04f31ccf94f51f6c01304f6729f861482963b7efR121) before uploading

Reply:
I meant what constraints do we need to enforce on these values, minimal string length, fixed values, etc?

We're using `coerce=True`, which from my reading would convert input values to target type first. Do we need this?

Reply:
Regarding the content (length, values etc) I think it's difficult to put on guards. The model locator/id objects/embedding string could each vary in size. Good question about `coerce=True`, tbh I just took this logic from the `LocatorEmbeddingsDataFrameSchema` above assuming there was some rationale behind it.